lib/board.rb

```diff
@@ -34,8 +34,48 @@ def free_positions
     board.flatten.select { |cell| cell.is_a?(Integer) }
   end
 
+  def string_to_board(board_string)
```

Comment:
How come this is an instance method? This means I need an instance of a board FIRST before I can fill it up with data from the string...